package.json

```diff
@@ -12,12 +12,16 @@
     "build": "npm run clean && npm run build-version && npm run build-lib && npm run build-types",
     "build-lib": "babel src -d lib --source-maps --extensions '.ts,.js'",
     "build-types": "tsc --emitDeclarationOnly",
-    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
+    "build-version": "sh ./timestamp.sh > src/versionInfo.ts  && eslint \"src/versionInfo.ts\" --fix",
     "watch": "npm run build-version && babel src --out-dir lib --watch --source-maps --extensions '.ts,.js'",
     "clean": "rm -rf lib",
-    "lint": "eslint 'src/**/*.js' 'src/**/*.ts'",
-    "lint-fix": "eslint 'src/**/*.js' 'src/**/*.ts' --fix",
-    "test": "npm run lint",
+    "coverage": "jest --coverage --collectCoverageFrom=src/**/*.[jt]s",
+    "lint": "eslint \"src/**/*.js\" \"src/**/*.ts\"",
+    "lint-fix": "eslint \"src/**/*.js\" \"src/**/*.ts\" --fix",
+    "jest": "jest",
```

Comment:
I don't think this really adds anything?

Reply:
It's needed if you want to run only the jest tests. I tried removing it but then I see:
```sh
~/gh/solid/solid-ui $ npm run jest
npm ERR! missing script: jest
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     test

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/michiel/.npm/_logs/2020-02-10T19_14_04_093Z-debug.log

Reply:
Try `npx jest`.